docs/antctl.md

```diff
@@ -626,3 +627,26 @@ Example outputs of record metrics:
 RECORDS-EXPORTED RECORDS-RECEIVED FLOWS EXPORTERS-CONNECTED
 46               118              7     2      
 ```
+
+### Multi-Cluster commands
+
+Starting with version 1.6.0, Antrea supports the `antctl mc` commands, which can collect information from multicluster cluster. 
+This information is useful when tryting to troubleshoot issues in
```

Comment:
s/tryting/trying/
s/in/in a ClusterSet/

Reply:
done